views/event_details.hbs

```diff
@@ -15,7 +15,7 @@
 
   <div class="information">
     <h3  class="text-danger">categories: </h3> <h4 class="text-info">{{event.categories}}</h4>
-    <h3  class="text-danger">price: </h3> <h4 class="text-info">{{event.cost}}</h4>
+    <h3  class="text-danger">price: </h3> <h4 class="text-info">{{event.cost}} <img class="shekelIcon" src="../imges/shekel.png" alt="shekel icon"></h4>
```

Comment:
the src is wrong. You obviously didn't actually test this....
Also, the shekel icon shouldn't be in this PR, but I'll let it slide this time.

Reply:
I tested it, it was working but I change the name of the file , it was imges not images